styles/main.css

```diff
@@ -291,29 +223,6 @@ a {
     padding-top: 95px;
```

Comment:
you should be able to just add padding-top to all gallery pages and not use gallery id for your about page